CRM/Civicase/Upgrader.php

```diff
@@ -398,6 +398,15 @@ private function getCaseMenuItem($name) {
     return $result['values'][0];
   }
 
+  /**
+   * Adds case lock table to existing installations.
+   */
+  public function upgrade_0001() {
```

Comment:
We already have `function upgrade_4200`. is the numbering correct?

Reply:
That is an example of an upgrader, not an actual one.